### PR TITLE
Time and Brightness unit metrics

### PR DESCRIPTION
**Pull request by fred2gombert**
Adding two new types of metrics: Time and Brightness

**Description (for Contributor and Core Developer)**

**Definition Of Done (for Core Developer only)**

| Q                                 | A
| --------------------------------- | ---
| Added Specs                       | No
| Added legacy Behats               | No
| Added acceptance tests            | No
| Added integration tests           | No
| Changelog updated                 | No
| Review and 2 GTM                  | No
| Micro Demo to the PO (Story only) | No
| Migration script                  | -
| Tech Doc                          | -

`Todo`: Pending / Work in progress
`OK`: Done / Validated
`-`: Not needed

### CODE REVIEW

**Comment by fred2gombert** [comment]
> I put my faith in you about the convert operations

Believe me, there is really 24 hours in one day ;)

File: src/Akeneo/Tool/Bundle/MeasureBundle/Resources/config/measure.yml [hunk 1]
```diff
@@ -476,3 +476,27 @@ measures_config:
...
+               symbol: yrs
+           DAY:
+               convert: [{‘mul’: 24}]
+               symbol: days
+           HOUR:
+               convert: [{‘mul’: 1}]
+               symbol: hrs
+           MINUTE:
+               convert: [{‘mul’: 0.01666666667}]
+               symbol: mins
```
Comment (mmetayer) [inline src/Akeneo/Tool/Bundle/MeasureBundle/Resources/config/measure.yml]: we already have a 'Duration' metric family, please update it instead if need be

File: src/Akeneo/Tool/Bundle/MeasureBundle/Resources/config/measure.yml [hunk 2]
```diff
@@ -476,3 +476,27 @@ measures_config:
             DOZEN:
                 convert: [{'mul': '12'}]
                 symbol: Dz
+    Brightness:
+        standard: LUMEN
+        units:
+            LUMEN:
+                convert: [{‘mul’: 1}]
```
Comment (pierallard) [inline src/Akeneo/Tool/Bundle/MeasureBundle/Resources/config/measure.yml]: NOPE
` is not '

File: src/Akeneo/Tool/Bundle/MeasureBundle/Resources/config/measure.yml (hunk 1 above)
Comment (fred2gombert) [reply src/Akeneo/Tool/Bundle/MeasureBundle/Resources/config/measure.yml]: indeed  - fixed

File: src/Akeneo/Tool/Bundle/MeasureBundle/Resources/config/measure.yml (hunk 2 above)
Comment (fred2gombert) [reply src/Akeneo/Tool/Bundle/MeasureBundle/Resources/config/measure.yml]: fixed

**Pull request merged**